artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -115,13 +115,13 @@ paths:
         "401":
           $ref: "#/components/responses/Generic401"
         "403":
-          $ref: "#/components/responses/CreateSubscription403"
+          $ref: "#/components/responses/CreateSubscriptionPermissionDenied403"
         "409":
           $ref: "#/components/responses/Generic409"
         "415":
           $ref: "#/components/responses/Generic415"
         "422":
-          $ref: "#/components/responses/CreateSubscription422"
+          $ref: "#/components/responses/Generic422"
         "429":
           $ref: "#/components/responses/Generic429"
         "500":
```

Comment:
How about other errorcodes in the guideline; e.g. 405,406,501 , I think those ones would make a sense here.

Reply:
I did not add them because they were not in the initial yaml.
Regarding 405 & 406 I'm fine to add them. @PedroDiez WDYT?
Regarding 501 with @PedroDiez we think that this API will "not work" if all operation are not implemented. Indeed, it you provide subscription you must provide POST/GET/DELETE

Reply:
Fine for me to discuss it later.